src/core/component.test.ts

```diff
@@ -886,6 +886,35 @@ describe("nested option", () => {
         });
     });
 
+    it("update position nested components by v-for", (done) => {
```

Comment:
```suggestion
    it("update nested components created by v-for", (done) => {
```

since test has no check for position